src/connectors/range/__tests__/connectRange-test.js

```diff
@@ -526,8 +526,11 @@ describe('connectRange', () => {
     const rendering = () => {};
     const createHelper = () => {
       const helper = jsHelper(fakeClient);
-      helper.clearRefinements = jest.fn();
       helper.search = jest.fn();
+      const initalClearRefinements = helper.clearRefinements;
+      helper.clearRefinements = jest.fn((...args) =>
```

Comment:
Why wrap the `clearRefinements` method?

Reply:
I wrap the `clearRefinements` method to be able to test if she was been called. You are right what matter is the result at the end. But we should check that the `clearRefinements` is called with the given attribute name in order to avoid to clear all the current refinements (see: #2459).

Reply:
Let's try that. If we don't like it we can revert back :)